assets/styl/_layout/_loading.styl

```diff
@@ -2,8 +2,7 @@
 //
 // _layout/_loading.styl
```

Comment:
Kss 3.x.x reference name of file (and line) on generated documentation.
So, write here is not necessary.